java/cli/src/main/java/org/ray/cli/RayCli.java

```diff
@@ -13,43 +13,42 @@
  */
 public class RayCli {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(RayCli.class);
+  private static final Logger logger = LoggerFactory.getLogger(RayCli.class);
```

Comment:
why lower-case this variable name?

Reply:
= =...

to unify all `logger` definition?